packages/cli/package.json

```diff
@@ -2,6 +2,13 @@
   "name": "@code-pushup/cli",
   "version": "0.49.0",
   "license": "MIT",
+  "description": "A CLI to run all kinds of code quality measurements to align your team with company goals",
+  "homepage": "code-pushup.dev",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-pushup/cli.git",
+    "directory": "packages/cli"
+  },
```

Comment:
This metadata is automatically handled by our build script - see [`esbuild.config.js`](https://github.com/code-pushup/cli/blob/main/esbuild.config.js#L99-L106). This way we can maintain most of the metadata in one place (the root `package.json`). Only the `description`s need be configured in the individual source `package.json`s. The `homepage` and `repository` fields can be auto-filled from root.

Reply:
Thanks for pointing that out!

Reply:
Done